metrics/issues-open.md

```diff
@@ -20,11 +20,11 @@ Mandatory:
 ### Bugzilla Case
 
 * Platform: The platform provides several statuses and those depends on the configuration of each project. By default Bugzilla has a [workflow](https://www.bugzilla.org/docs/3.6/en/html/lifecycle.html) where the open statuses may be considered: all but *closed* and *resolved*.
-* Case Definition: Open issues need the statuses parameter to be defined. By default this may be all statuses but *closed* and *resolved*. 
+* Case Definition: Open issues need the statuses parameter to be defined. By default, this may be all statuses, except for  *closed* and *resolved*. 
 
 ## 2. Use Cases
 
-* Many people would like an at-a-glance visualization of all the open issues in a project, this visualization would provide that without having to look inside every sub-project in the repository and sum the issues manually
+* Many people would like an at-a-glance visualization of all the open issues in a project. This visualization would provide that without having to look inside every sub-project in the repository and sum the issues manually

```

Comment:
I am not too sure about what is meant by this particular use case, and hence, not confident about this change. Please suggest a better change if the current one deviates from the meaning of the paragraph.

Reply:
This change is ok, thanks.

Reply:
Seems good to me.